src/content/deployment/subctl.en.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "subctl"
+weight: 10
+---
+
+`subctl` is a command line utility designed to simplify the deployment and maintenance of
+Submariner across your clusters.
+
+### Synopsis
+
+`subctl [command] [--flags] ...`
+
+### Description
+
+`subctl` helps to automate the deployment of the Submariner [operator](https://github.com/submariner-io/submariner-operator) thereby reducing the possibility of mistakes during the process.
+
+`subctl` connects to specified cluster(s) and performs the requested *command*. 
+
+### Commands
+
+#### deploy-broker
+
+`subctl deploy-broker [flags]`
+
+The **deploy-broker** command configures the cluster specified by the `--kubeconfig` flag
+(or `KUBECONFIG` env var) and the `--kubecontext` flag as the Broker. It installs
+the necessary CRDs and the `submariner-k8s-broker` namespace.
+
+In addition, it generates a `broker-info.subm` file which can be used with the `join` command
+to connect clusters to the Broker. This file contains the following details:
+
+ * Encryption PSK key
+ * Broker access details for subsequent `subctl` runs.
+ * Service discovery settings
+ * Globalnet settings
+
+##### deploy-broker flags
+
+    -h, --help                           help for deploy-broker
+
+    --kubeconfig <string>                  absolute path(s) to the kubeconfig file(s) (default "$HOME/.kube/config")
+
+    --kubecontext <string>                 kubeconfig context to use
+
+    --service-discovery                  Enable Multi Cluster Service Discovery
+
+    --globalnet                          Enable support for Overlapping cluster/service CIDRs in connecting clusters (default disabled)
+
+    --globalnet-cidr-range <string>        Global CIDR supernet range for allocating GlobalCIDRs to each cluster
+                                         (default "169.254.0.0/16")
+
+    --ipsec-psk-from <string>              Import IPSEC PSK from existing submariner broker file, like broker-info.subm
+                                         (default "broker-info.subm)
+
+    --dataplane                          Install the Submariner dataplane on the broker
+                                         If this flag is enabled, the broker will be joined as if a join command 
```

Comment:
Last time I tried --dataplane, it did not work and all our quickstart guides use separate commands for deploy-broker.
Shall we remove this flag from deploy-broker?